src/main/java/com/internship/backend/model/DateRangeRequest.java

```diff
@@ -0,0 +1,16 @@
+package com.internship.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Setter
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class DateRangeRequest {
```

Comment:
Could you check if this class could be rewritten as a record? https://www.baeldung.com/java-record-keyword